gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.1-all.zip
```

Comment:
Bumping this isn't a bad idea, but I tend to keep it on the least supported version so that we don't accidently include gradle API that would break compatibility with 8.4.

Reply:
But it is a minor version, not a major one. Best way to go would be to just bump FG to 6.1 maybe?

Reply:
If we drop support then yes it would require a version bump on our end. Which honestly I dont think is worth it. There is no compelling reason to bump gradle.

Reply:
Rolled back Gradle in latest commit